btcjson/v2/btcjson/chainsvrresults.go

```diff
@@ -6,6 +6,11 @@ package btcjson
 
 import "encoding/json"
 
+// GenerateResult models the data returned from the generate command.
+type GenerateResult struct {
+	Blockhashes []string `json:"blockhashes"`
```

Comment:
This type isn't needed (or correct).  The result type of the `generate` RPC should just be a primitive JSON array.  By putting it into a struct like this, it would result in a json object like so:
`{'blockhashes': ["hash1","hash2"]}`

Instead of the desired:

`["hash1","hash2"]`